control_allocation_node:
  ros__parameters:
    arm_length: 0.229  # meters
    k_thrust: 8.54858e-06  # N/(rad/s)^2
    k_drag_to_thrust: 0.016  # N·s/m # Moment Constant in the plugin.
    motor_directions: [-1, 1, -1, 1, -1, 1]  # Motor direction for each motor, 1 for clockwise, -1 for counter-clockwise
    max_motor_speed: 1000.0  # rad/s
    min_motor_speed: 0.0  # rad/s
    max_tilt_angle: 1.57  # radians (30 degrees)
    min_tilt_angle: -1.57  # radians (-30 degrees)

hover_openloop_control_node:
  ros__parameters:
    gravity: 9.8  # m/s²
    mass: 3.646  # kg

other_params:
  ros__parameters:
    inertia: [0.04092, 0.04019, 0.06921, 0.00005656, -0.00006494, 0.00001313]  # kg*m² ((Ixx, Iyy, Izz, Ixy, Ixz, Iyz))